crt_portal/cts_forms/forms.py

```diff
@@ -1400,7 +1401,15 @@ def get_actions(self, report):
             if field in ['summary', 'comment']:
                 continue
             initial = getattr(report, field, 'None')
-            yield f"{name}:", f'Updated from "{initial}" to "{updates[field]}"'
+
+            def field_changed(old, new):
```

Comment:
Does this need to be or is there a benefit to having this defined within the `for` loop? It doesn't look like we're accessing anything outside of the `field_changed` context

Reply:
Nope, I'll move it outside the loop - thanks!